Makefile

```diff
@@ -78,8 +78,10 @@ create-awsfederatedaccountaccess: check-aws-account-id-env
 
 .PHONY: test-switch-role
 test-switch-role:
+	# Retrieve role UID
+	$(eval UID=$(shell oc get awsfederatedaccountaccesses.aws.managed.openshift.io  test-federated-user -o=json |jq -r .metadata.labels.uid))
```

Comment:
Is there a reason we use the full domain for this?  Just curious.

Reply:
The CRD has it as a valid name, and I think that was just a tab-complete from the command I was using.  Both the full one and `awsfederatedaccountaccess` works.  I don't have a preference for which to use, though.